### PR TITLE
MDP-198:Article page component

### PR DESCRIPTION
**Pull request by kolesarovakristina**

### CODE REVIEW

File: src/components/ArticlePage/Articles.tsx
Comment (pggl) [inline src/components/ArticlePage/Articles.tsx]: why is article array? there will be only one article per page

Comment (pggl) [reply src/components/ArticlePage/Articles.tsx]: and what is inside should be Article component

File: src/components/ArticlePage/Articles.tsx
Comment (pggl) [inline src/components/ArticlePage/Articles.tsx]: wrong name for component

File: src/components/Testing.tsx
Comment (pggl) [inline src/components/Testing.tsx]: revert all this changes, it is only for development

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/static/svg/group-13.svg
Comment (pggl) [inline src/static/svg/group-13.svg]: wrong name for this svg, what is group-13.svg?

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/components/ArticlePage/ArticleComponent.tsx
Comment (garySiladi) [inline src/components/ArticlePage/ArticleComponent.tsx]: can you rename this to something like `ArticleWrapper`

**Review by garySiladi — CHANGES_REQUESTED** [review]
.

File: src/components/ArticlePage/ArticleWrapper.tsx
Comment (garySiladi) [inline src/components/ArticlePage/ArticleWrapper.tsx]: rename this to something like `ArticleContainer`

**Review by garySiladi — CHANGES_REQUESTED** [review]

File: src/components/ArticlePage/ArticleContainer.tsx
Comment (garySiladi) [inline src/components/ArticlePage/ArticleContainer.tsx]: you should import this as an external component. are you waiting on your landing page PR to be merged, and will reuse from there?

Comment (kolesarovakristina) [reply src/components/ArticlePage/ArticleContainer.tsx]: When landing page will be merged I change this.

**Review by garySiladi — DISMISSED** [review]
ok from my side just update with the component from MDP-185 once that's merged

File: src/components/ArticlePage/Article.tsx
Comment (pggl) [inline src/components/ArticlePage/Article.tsx]: if this is title there should be h1 tag

File: src/components/ArticlePage/Article.tsx
Comment (pggl) [inline src/components/ArticlePage/Article.tsx]: and this should be h2 tag

File: src/components/ArticlePage/Article.tsx
Comment (pggl) [reply src/components/ArticlePage/Article.tsx]: and there is also special article tag https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article

File: src/components/ArticlePage/ArticleContainer.tsx
Comment (pggl) [inline src/components/ArticlePage/ArticleContainer.tsx]: we don't need useState

**Review by pggl — CHANGES_REQUESTED** [review]

File: src/components/ArticlePage/Article.tsx
Comment (pggl) [inline src/components/ArticlePage/Article.tsx]: there should be only one article tag

**Review by pggl — CHANGES_REQUESTED** [review]

**Review by pggl — APPROVED** [review]

**Pull request merged**
